### DISCUSSION TITLE
有人去过六分街的这个录像店吗？

### DISCUSSION THREAD

**Discussion by KEARUM** [share121/inter-knot#612] [其他]
![20240810122546](https://github.com/user-attachments/assets/bfe05d14-f2a1-4bb1-abfb-eb533be77082)

听同学说这里的片子超全的诶，而且店长好像还是个很年轻的兄妹？
有没有去过的啊？说一下这家店如何？

**Comment by Tragedysaver1024** [comment]
服务态度十分的好，除了有一回柜员好像换成了维多利亚家政的一名工作人员，态度有点不耐烦……
除此之外还是挺好的，甚至还有那种特别攒劲的节目能租

**Reply by 1155993355** [reply]
盲猜那个工作人员是艾莲

**Comment by kaijinti** [comment]
只感觉文艺片很多

**Comment by lllliufeng** [comment]
没有人比他们更懂录像带

**Comment by KEARUM** [comment]
这样的吗？那我找时间也去看一看！正好最近没什么新片子看了！

**Comment by WORLD-executeME** [comment]
超级推荐，两位店长很有品味

**Comment by lingyushen** [comment]
<link rel="stylesheet" href="path-to-css-file">
<h1>经常看到店长带人进卧室</h1>
 h1 {
    color: #39c5bb;
    font-size: 4rem;
}

**Reply by keerdun** [reply]
这样的超大字体怎么打的啊？

**Comment by GanlinSama** [comment]
wok，六楼消息这么劲爆么？难道说他们家还干着灰色产业？

**Comment by GWWA** [comment]
楼上小心些，据说这家店主也是绳匠，小心背刺！

**Comment by FIREFLY26710-WEB** [comment]
想要店长小姐的联系方式！

**Comment by Buer-Nahida** [comment]
# 去过，录像带倒是挺全的，但不建议去那里做兼职，我每天去一次，每天都是同一个服务员

**Comment by Nina123973** [comment]
嗯呢嗯呢

**Comment by ZhuYuan-148** [comment]
店长人还是很好的，我有关于录像带的帮助都会去找店长。

**Comment by Larksonguesinaspic** [comment]
挺好的
